bot/cogs/guess_number.py

```diff
@@ -0,0 +1,100 @@
+import discord
+import math, random
+from discord.ext.commands import Bot, Cog, command
+
+
+class guess_game(Cog): #(Cog): # Include Cog if it's a part of Cog file in Friendo Bot. Change class name if needed.
+    """Commands for Guessing Right Number"""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    # Variables in class guess_game
+    #Default bounds, random number, and rounds.
+    lower_bound = 1
+    upper_bound = 10
+
+    check_number = round(random.randint(lower_bound, upper_bound))
+    rounds_number = round(math.log(upper_bound - lower_bound + 1,
+                                   2))
+
+    # Look in `guess` command below.
```

Comment:
Not a very helpful comment. This can be removed, or changed to something a bit more descriptive.